<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>594</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>776</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/app.png</normaloff>../icons/app.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: transparent;
}

#controlPanel, #container {
	background-color: #ADD8DA;
	border-radius: 26%;
	border-bottom: 3px outset #79BFC3;
	border-right: 3px outset #79BFC3;
}

#previewContainer {
	border-radius: 10%;
	background-color: qlineargradient(spread:pad, x1:0.261, y1:0.739045, x2:0.727273, y2:0.284, stop:0 rgba(242, 197, 124, 255), stop:1 rgba(207, 169, 106, 255));
	border-bottom: 2px solid #D1AA6B;
	border-right: 3px solid #D1AA6B;
}

QComboBox {
	background-color: #4E4187;
	color: #ADD8DA;
}

QComboBox QListView {
	color: #ADD8DA;
}


QLabel, QPushButton {
	color: #4E4187;
	font-family: Arial, Helvetica, sans-serif;
}

#urlInput {
	border: none;
	background-color: #4E4187;
	color: #ADD8DA;
}

#text1 {
	border: 1px dashed black;
}


#rightSide QPushButton {
	background-color: #DB2B39;
	color: #9c0d38;
	border-radius: 7%;
	border-bottom: 2px solid #9c0d38;
}

#rightSide QPushButton::hover {
	background-color: #B32430;
}

#rightSide QPushButton:pressed {
	color: #A54D68;
	background-color: #B66F84;
    border-color:#91707A;
    border-style: inset;
}

#returnButton {
	background-color: blue;
	border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="controlPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="pageLogo">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../icons/arrow-down.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pageName">
         <property name="font">
          <font>
           <family>Arial,Helvetica,sans-serif</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Download page</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>343</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="minimize">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/delete.png</normaloff>../icons/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/cross.png</normaloff>../icons/cross.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="container">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="previewContainer">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="videoPreview">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>240</width>
              <height>135</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../icons/previewTest.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="parseInfo">
            <item>
             <widget class="QLabel" name="channelLogo">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../icons/authorizationLogo.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="videoInfo">
              <item>
               <widget class="QLabel" name="videoName">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial,Helvetica,sans-serif</family>
                 </font>
                </property>
                <property name="text">
                 <string>СОКРОВИЩНЫЙ ПЕРЕДОЗ | Мультивселенная Острова Сокровищ</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="channelName">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial,Helvetica,sans-serif</family>
                 </font>
                </property>
                <property name="text">
                 <string>- Sndk</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightSide">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="returnLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="returnButton">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../icons/arrow-left.png</normaloff>../icons/arrow-left.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="text1">
            <property name="font">
             <font>
              <family>Arial,Helvetica,sans-serif</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Paste the video url, make sure that it's valid and download it by selecting the quality and format that you need =)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="urlLayout">
            <item>
             <widget class="QLabel" name="text2">
              <property name="font">
               <font>
                <family>Arial,Helvetica,sans-serif</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>URL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="urlInput">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="qualityLayout">
            <item>
             <widget class="QLabel" name="text3">
              <property name="font">
               <font>
                <family>Arial,Helvetica,sans-serif</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Quality:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="qualityInput">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>1440p</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1080p</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>720p</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>480p</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>360p</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="text4">
              <property name="font">
               <font>
                <family>Arial,Helvetica,sans-serif</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>or lower (if not available)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="formatLayout">
            <item>
             <widget class="QLabel" name="text5">
              <property name="font">
               <font>
                <family>Arial,Helvetica,sans-serif</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>mp4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mp3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>avi</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="errText">
            <property name="font">
             <font>
              <family>Arial,Helvetica,sans-serif</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="sizeText">
            <property name="font">
             <font>
              <family>Arial,Helvetica,sans-serif</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Estimated size: 0.00GB</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="bottomButtons">
            <item>
             <widget class="QPushButton" name="addToQueueButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial,Helvetica,sans-serif</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add to queue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="downloadButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial,Helvetica,sans-serif</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Download</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
